src/components/loading-indicator/LoadingIndicator.tsx

```diff
@@ -16,7 +16,7 @@ export interface LoadingIndicatorProps {
 }
 
 const LoadingIndicator = ({ className = '', size = LoadingIndicatorSize.DEFAULT }: LoadingIndicatorProps) => (
-    <div className={`crawler ${className} is-${size}`}>
+    <div className={`crawler ${className} is-${size}`} data-testid="loading-indicator">
```

Comment:
can we use the blueprint loading indicator instead of the buie one? it should have a role on the blueprint loading indicator that can be targeted

Reply:
I think we can undo this change since you swapped to the blueprint component